src/api/base_method.yaml

```diff
@@ -0,0 +1,25 @@
+namespace: "methods"
+info:
+  type: method
+  type_info:
+    label: Method
+    summary: A label projection method.
+    description: |
+      A label projection method to predict the labels of a new "test"
+      dataset based on an annotated "training" dataset.
+arguments:
+  - name: "--input_train"
+    __merge__: file_train.yaml
+    direction: input
+    required: true
+  - name: "--input_test"
+    __merge__: file_test.yaml
+    direction: input
+    required: true
+  - name: "--output"
+    __merge__: file_prediction.yaml
+    direction: output
+    required: true
+test_resources:
+  - type: python_script
+    path: /common/component_tests/run_and_check_output.py
```

Comment:
This is the wrong check, `check_config` should go in this file and `run_and_check_output` should go in `comp_method.yaml`. Sorry I missed this before.